jobs/build/appregistry/Jenkinsfile

```
@@ -145,10 +146,36 @@ node {
                     appregistry.stageAttachMetadata(operatorBuilds)
                 }
             }
+            stage("slack notification to release channel") {
+                if (params.STREAM != 'stage' || params.STREAM != 'prod') {
```

this will always skip the notif. how about:
```suggestion
                if (params.STREAM == 'dev') {
```